shinhye/app.js

```diff
@@ -0,0 +1,33 @@
+const express = require("express");
+const cors = require("cors");
+const app = express();
+const dotenv = require("dotenv");
+const morgan = require('morgan');
+
+dotenv.config();
+
+app.use(cors());
+app.use(morgan('combined'));
+
+const { DataSource } = require("typeorm");
+
+const myDataSource = new DataSource({
+  type: process.env.TYPEORM_CONNECTION,
+  host: process.env.TYPEORM_HOST,
+  port: process.env.TYPEORM_PORT,
+  username: process.env.TYPEORM_USERNAME,
+  password: process.env.TYPEORM_PASSWORD,
+  database: process.env.TYPEORM_DATABASE,
+});
+
+myDataSource.initialize().then(() => {
+  console.log("Data Source has been initialized!");
+});
+
+app.get("/ping", function (req, res, next) {
+  res.json({message: "pong"});
+});
```

Comment:
1. function 키워드 함수 vs 화살표 함수 차이
2. 함수가 종료되는 구문에서는 return 키워드 사용

```suggestion
app.get("/ping", (req, res) => {
  return res.json({message: "pong"});
});
```

Reply:
확인 후 반영했습니다 감사합니다!